server/features/search_combined.feature

```diff
@@ -3,7 +3,7 @@ Feature: Search Events and Planning
         Given "agenda"
         """
             [
-                {"name": "sports", "_id": "sports", "is_enabled": true},
+                {"name": "sports", "_id": "68345becf63aa4e3993ff94d", "is_enabled": true},
```

Comment:
Rather than specifying the IDs manually, we can now use then with expansion. In this case
`"#agenda_0._id#"` Would be the ID for Sports Agenda
`"#agenda_1._id#"` Would be the ID for Finance Agenda
`"#agenda_2._id#"` Would be the ID for Entertainment Agenda

Reply:
See https://github.com/superdesk/superdesk-core/blob/async/superdesk/tests/steps.py#L524 on how it's added